src/message/messageActionSheet.js

```diff
@@ -89,6 +90,10 @@ const editMessage = async ({ message, actions }: MessageAuthAndActions) => {
   actions.startEditMessage(message.id);
 };
 
+const deleteMessage = async ({ auth, message }: AuthAndMessageType) => {
+  patchMessage(auth, '(deleted)', message.id);
```

Comment:
`patchMessage(auth, '' , message.id)` 
This too worked. I just thought not to leave the empty strings.